api/graphql/mutation/calendarEventCreate/type.graphql

```diff
@@ -5,6 +5,8 @@ input CalendarEventCreateInput {
   start_at: DateTime!
   end_at: DateTime!
   location: String
+  is_open: Boolean!
```

Comment:
No need to have them as required fields. By default they are both false.

Reply:
Done!